aws/data_source_aws_vpc_peering_connection.go

```diff
@@ -40,6 +40,19 @@ func dataSourceAwsVpcPeeringConnection() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"cidr_block_set": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
Since the attribute is not used for configuring the data source, `Optional` should be removed. 👍 

```suggestion
```